src/app/views/onboarding/step2.html

```diff
@@ -7,34 +7,37 @@ <h1 translate>mno_enterprise.templates.onboarding.select_your_app.title</h1>
         </div>
         <div class="apps-container">
           <!-- Left column: App icons -->
-          <mno-widget id="apps-widget" is-loading="vm.isLoading">
-            <mno-widget-header>
-              <div id="widget-header-bar">
-                <div id="app-search" class='input-group'>
-                  <div class="input-group-addon"><span class="glyphicon glyphicon-search"></span></div>
-                  <input type="text" ng-model="vm.searchTerm" placeholder="{{ 'mno_enterprise.templates.onboarding.select_your_app.search' | translate }}" class="form-control" ng-change="vm.selectedCategory = ''">
-                </div>
-                <select id="app-category" class="form-control" ng-model="vm.selectedCategory" ng-options="c for c in vm.marketplace.categories" ng-change="vm.searchTerm = ''">
-                  <option value=''>{{ 'mno_enterprise.templates.dashboard.marketplace.index.select_all' | translate }}</option>
-                </select>
-              </div>
-            </mno-widget-header>
-            <mno-widget-body class="no-padding large">
-              <div class="app-tile" ng-repeat="app in vm.marketplace.apps | filter:vm.appsFilter | filter:{name: vm.searchTerm}">
-                <a class="app-info-modal" href="" ng-click="vm.openInfoModal(app)"><i class="fa fa-info-circle"></i></a>
-                <div class="app-icon" ng-click="vm.toggleApp(app)">
-                  <div class="checked" ng-show="app.checked">
-                    <i class="fa fa-check fa-5x"></i>
+          <div id="apps">
+            <section>
```

Comment:
I think you can remove this `section` and merge its LESS instructions (if any) in `.apps`. This is to keep the DOM simple. Usually, you have one container and its elements right under it.
If you have a good reason to use it, keep it of course.